release/kata-deploy-binaries.sh

```diff
@@ -89,18 +93,28 @@ install_image() {
 
 #Install kernel asset
 install_kernel() {
-	go get "github.com/${project}/packaging" || true
-	(
-		cd ${GOPATH}/src/github.com/${project}/packaging >>/dev/null
-		git checkout "${kata_version}-kernel-config" ||
-		git checkout "${kata_version}"
-
+	if [[ "$test_local" == "true" ]]; then
+		pushd ${GOPATH}/src/github.com/${project}/packaging
 		info "build kernel"
 		./kernel/build-kernel.sh setup
 		./kernel/build-kernel.sh build
 		info "install kernel"
 		DESTDIR="${destdir}" PREFIX="${prefix}" ./kernel/build-kernel.sh install
-	)
+		popd
+	else
+		go get "github.com/${project}/packaging" || true
+		(
```

Comment:
Nit: From a consistency point of view, it might be better to use `pushd`/`popd` as used in the first part of the `if` test above?

Reply:
Agree. Also, we could pull out some of the common steps out of the loop.

Reply:
changes applied

Reply:
Thanks @GabyCT. I think you can remove the brackets on lines 106 and 117 now that you're using `pushd`/`popd`.